### PR TITLE
✨ New Product: Sim4Life Engineering, coming soon! (🚨)

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

New Product: Sim4Life Engineering, coming soon!

This PR brings the new ``s4lengine`` product to oSPARC. For now we will only displayed a blurred login page with a "Coming Soon" banner on top of it.

🚨
- Add new product ``s4lengine`` to DB

image

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by YuryHrytsuk — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]

File: services/web/server/src/simcore_service_webserver/statics/plugin.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def setup_statics(app: web.Application) -> None:
     setup_products(app)
 
     # serves information composed by making 4 http requests (once for each product)
-    # to the index.html in each of the 4 product directories /osparc, /s4l, /s4llite, /s4lacad, /s4ldesktop, /s4ldesktopacad,  and /tis
+    # to the index.html in each of the 4 product directories /osparc, /s4l, /s4llite, /s4lacad, /s4lengine, /s4ldesktop, /s4ldesktopacad, and /tis
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/statics/plugin.py:34]: *N* product directories :-)

File: services/web/server/src/simcore_service_webserver/statics/_events.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
 
 async def create_cached_indexes(app: web.Application) -> None:
     """
-    Currently the static resources are contain 4 folders: osparc, s4l, s4llite, s4lacad, tis
+    Currently the static resources are contain 4 folders: osparc, s4l, s4llite, s4lacad, s4lengine, tis
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/statics/_events.py:51]: **N** folders

**Review by pcrespov — APPROVED** [review]

**Review by ignapas — APPROVED** [review]

**Review by GitHK — COMMENTED** [review]
👍

**Review by jsaq007 — APPROVED** [review]
👍🏼

**Review by GitHK — APPROVED** [review]

**Pull request merged**